src/utils/openLinkProgrammatically.ts

```diff
@@ -1,13 +1,19 @@
-import { Browser } from "@capacitor/browser";
 import { Capacitor } from "@capacitor/core";
+import { AppLauncher } from '@capacitor/app-launcher';
 
-// Safari is a jerk and is aggressive about blocking window.open calls if the logic is too complex
-// Capacitor should be doing this for us but \o/
-export function openLinkProgrammatically(href?: string) {
-    if (!href) return;
+export async function openLinkProgrammatically(url?: string) {
+    if (!url) return;
     if (Capacitor.isNativePlatform()) {
-        Browser.open({ url: href || "", windowName: "_blank" });
+        const { value } = await AppLauncher.canOpenUrl({ url });
+
+        if (!value) {
+            // Try to open in browser just in case that works glhf
+            window.open(url || "", "_blank");
+            return;
```

Comment:
Actually should probably throw an error or something instead? On iOS I get nothing visual displayed when I don't have a nostr client installed. Just these error logs: 

```
⚡️  To Native ->  AppLauncher canOpenUrl 37902078
⚡️  TO JS {"value":false}
Failed to open URL nostr+walletconnect://d8f8dd6eede3b8b6ed758797f0f463925382c004711f58ae07ff8f6da0b4629f?relay=wss%3A%2F%2Fnostr.mutinywallet.com%2F&secret=7d4041df706206caa145882ac56b3da550d84d67fd5d25358fa5f3a791590c9b: Error Domain=NSOSStatusErrorDomain Code=-10814 "(null)" UserInfo={_LSLine=268, _LSFunction=-[_LSDOpenClient openURL:fileHandle:options:completionHandler:]}
```

Reply:
okay this should be good now